genericMLE Ubuntu test failure

(Note: quantal 64 doesn't show any errors or failures
https://launchpadlibrarian.net/144807901/buildlog_ubuntu-quantal-amd64.statsmodels_0.5.0~ppa17~revno-1430~quantal1_UPLOADING.txt.gz)

https://launchpadlibrarian.net/144804522/buildlog_ubuntu-raring-i386.statsmodels_0.5.0~ppa17~revno-1430~raring1_UPLOADING.txt.gz

and the same test error as in #967

```
======================================================================
FAIL: statsmodels.miscmodels.tests.test_generic_mle.TestMyPareto1.test_params
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/buildd/statsmodels-0.5.0~ppa17~revno/debian/python-statsmodels/usr/lib/python2.7/dist-packages/statsmodels/miscmodels/tests/test_generic_mle.py", line 82, in test_params
    assert_allclose(self.res1.bse, self.res1.bsejac, rtol=0.05, atol=0.15)
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 1179, in assert_allclose
    verbose=verbose, header=header)
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 608, in assert_array_compare
    chk_same_position(x_isnan, y_isnan, hasval='nan')
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 588, in chk_same_position
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=0.05, atol=0.15

x and y nan location mismatch:
 x: array([  2.86042918e-01,   4.47801961e-06,   8.06451325e-06])
 y: array([ 0.25885202,         nan,         nan])
```

failure with different numbers:

https://launchpadlibrarian.net/144805183/buildlog_ubuntu-quantal-i386.statsmodels_0.5.0~ppa17~revno-1430~quantal1_UPLOADING.txt.gz

```
======================================================================
FAIL: statsmodels.miscmodels.tests.test_generic_mle.TestMyPareto1.test_params
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/buildd/statsmodels-0.5.0~ppa17~revno/debian/python-statsmodels/usr/lib/python2.7/dist-packages/statsmodels/miscmodels/tests/test_generic_mle.py", line 82, in test_params
    assert_allclose(self.res1.bse, self.res1.bsejac, rtol=0.05, atol=0.15)
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 1168, in assert_allclose
    verbose=verbose, header=header)
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 636, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=0.05, atol=0.15

(mismatch 100.0%)
 x: array([  2.86042918e-01,   4.47801961e-06,   8.06451325e-06])
 y: array([ 0.29474762,  0.10073403,  0.25655678])

----------------------------------------------------------------------
```

--- a/statsmodels/miscmodels/tests/test_generic_mle.py
+++ b/statsmodels/miscmodels/tests/test_generic_mle.py
@@ -79,10 +79,12 @@ class CheckGenericMixin(object):
         assert_allclose(params, np.zeros(len(params)), atol=4)
 
         assert_allclose(self.res1.bse, np.zeros(len(params)), atol=0.5)
-        assert_allclose(self.res1.bse, self.res1.bsejac, rtol=0.05, atol=0.15)
-        # bsejhj is very different from the other two
-        # use huge atol as sanity check for availability
-        assert_allclose(self.res1.bsejhj, self.res1.bsejac,
+        if not self.skip_bsejac:
+            assert_allclose(self.res1.bse, self.res1.bsejac, rtol=0.05,
+                            atol=0.15)
+            # bsejhj is very different from the other two
+            # use huge atol as sanity check for availability
+            assert_allclose(self.res1.bsejhj, self.res1.bsejac,
                             rtol=0.05, atol=1.5)
 
 
@@ -109,6 +111,10 @@ class TestMyPareto1(CheckGenericMixin):
 
         self.mod = mod_par
         self.res1 = mod_par.fit(disp=None)
+
+        # Note: possible problem with parameters close to min data boundary
+        # see issue #968
+        self.skip_bsejac = True
 
     def test_minsupport(self):
         # rough sanity checks for convergence
@@ -140,3 +146,6 @@ class TestMyParetoRestriction(CheckGenericMixin):
 
         self.mod = mod_par
         self.res1 = mod_par.fit(disp=None)
+
+        # Note: loc is fixed, no problems with parameters close to min data
+        self.skip_bsejac = False

